tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -154,27 +173,71 @@ def test_brick_evict_with_node_down(self):
                 brick_id = node.get('id')
                 break
 
-        # Bring down the glusterfs node
-        vm_name = node_ops.find_vm_name_by_ip_or_hostname(hostname)
-        self.addCleanup(
-            self._wait_for_gluster_pod_after_node_reboot, hostname)
-        self.addCleanup(node_ops.power_on_vm_by_name, vm_name)
-        node_ops.power_off_vm_by_name(vm_name)
-
-        # Wait glusterfs node to become NotReady
-        custom = r'":.status.conditions[?(@.type==\"Ready\")]".status'
-        for w in waiter.Waiter(300, 20):
-            status = openshift_ops.oc_get_custom_resource(
-                self.ocp_client, 'node', custom, hostname)
-            if status[0] in ['False', 'Unknown']:
-                break
-        if w.expired:
-            raise exceptions.ExecutionError(
-                "Failed to bring down node {}".format(hostname))
+        self._bring_down_node_and_wait_to_become_notready(hostname)
 
         # Perform brick evict operation
         try:
             heketi_ops.heketi_brick_evict(h_node, h_server, brick_id)
         except AssertionError as e:
             if ('No Replacement was found' not in six.text_type(e)):
                 raise
+
+    @pytest.mark.tier4
+    def test_brick_evict_with_node_down(self):
+        """Test brick evict basic functionality and verify brick evict
+        will success after one node down out of more than three nodes"""
+
+        h_node, h_server = self.heketi_client_node, self.heketi_server_url
+
+        # Create heketi volume
+        vol_info = heketi_ops.heketi_volume_create(
+            h_node, h_server, 1, json=True)
+        self.addCleanup(
+            heketi_ops.heketi_volume_delete,
+            h_node, h_server, vol_info.get('id'))
+
+        # Get node on which heketi pod is scheduled
+        heketi_pod = openshift_ops.get_pod_name_from_dc(
+            self.ocp_client, self.heketi_dc_name)
+        heketi_node = openshift_ops.oc_get_custom_resource(
+            self.ocp_client, 'pod', '.:spec.nodeName', heketi_pod)[0]
+
+        # Get brick id and glusterfs node which is not heketi node
+        for node in vol_info.get('bricks', {}):
+            node_info = heketi_ops.heketi_node_info(
+                h_node, h_server, node.get('node'), json=True)
+            hostname = node_info.get('hostnames').get('manage')[0]
+            if (hostname != heketi_node):
```

Comment:
`if(hostname != heketi_node:` is sufficient

Reply:
Yes, as we don't here as we can pick any host on which bricks are hosted but heketi node.

Reply:
Done